# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/modules/Region.sys.mjs
# Commit: 1cf92ef74fb4
# Full Hash: 1cf92ef74fb45d3f5b62eb3f7b0fd7c35a2d4f4f
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-05-11 04:06:39
# Regressor Bug: 1810421
# File Overlap Count: 2
# Description:
#   Bug 1810421: Only scan wifi when the network changes or when on mobile r=necko-reviewers,kershaw
#   
#   Simplifies the concurrent operations of wifi scanning and reduces the
#   frequency in common cases.  Wifi scanning when on mobile is reduced from
#   every 5 seconds to every minute.  Wifi scans will also happen whenever
# ==============================================================================

diff -r afd62e82fe83 -r 1cf92ef74fb4 toolkit/modules/Region.sys.mjs
--- a/toolkit/modules/Region.sys.mjs	Wed May 10 19:46:29 2023 +0000
+++ b/toolkit/modules/Region.sys.mjs	Wed May 10 19:46:29 2023 +0000
@@ -764,7 +764,7 @@
     this.wifiService = Cc["@mozilla.org/wifi/monitor;1"].getService(
       Ci.nsIWifiMonitor
     );
-    this.wifiService.startWatching(this);
+    this.wifiService.startWatching(this, false);
 
     return new Promise(resolve => {
       this._wifiDataPromise = resolve;
